nautobot/core/settings.py

```diff
@@ -178,7 +158,7 @@
     ),
     "DEFAULT_VERSION": REST_FRAMEWORK_VERSION,
     "DEFAULT_VERSIONING_CLASS": "rest_framework.versioning.AcceptHeaderVersioning",
-    "PAGE_SIZE": PAGINATE_COUNT,
+    "PAGE_SIZE": None,
```

Comment:
The side effect of this will be disabling pagination in all of the places our bespoke `EnhancedPaginator` isn't used. Makes me think we might still consider some sane default here, even if it's hard-coded.

Reply:
Disregard. Thinking aloud is a victimless crime.